### ISSUE TITLE
Distributing APR-ImageJ via ImageJ update site

### ISSUE THREAD

**Issue by haesleinhuepf** [AdaptiveParticles/APR-ImageJ#1]
Hey guys,

I tried to play with the library and failed building it. Would it possible to distribute APR via ImageJ update site? Actually, I just want to load and save image stacks as APR... Nothing fancy.

Thanks!

Cheers,
Robert

**Comment by krzysg** [comment]
That is a plan for a (near) future. 
Personally I have never deployed any multi-OS (I guess we still need to provided precompiled libs for windows/linux/macos) jar and first I need to read somewhere what is the best way to do that... or find someone who did it. 
But again this is definitely what we need soon.

**Comment by imagejan** [comment]
FYI, the ImageJ updater respects platform-specific subfolder names of `./Fiji.app/jars/` and `./Fiji.app/lib/`.

For examples, you can have a look at the Java-8 and FFMPEG update sites:

* https://sites.imagej.net/Java-8/
* http://fiji.sc/~schindelin/ffmpeg-plugins/lib/

**Comment by skalarproduktraum** [comment]
@krzysg: If you have time next week I'm happy to help with the crossplatform stuff 👍

**Comment by haesleinhuepf** [comment]
@krzysg @skalarproduktraum I would be happy to help as well! Shall we schedule a mini-hackathon on this?

**Comment by krzysg** [comment]
Yes, definitely! I will send mail to you guys so we can decide when&where.

**Comment by vruetten** [comment]
Hi,
I also tried to build it with the following and ran into the error posted below. Any ideas as to how I could fix it?
- Maven home: /Users/../packages/apache-maven-3.6.1
- Java version: 12, vendor: Oracle Corporation

> Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Failure to find adaptiveparticles:libapr-java-wrapper:jar:0.1.0-SNAPSHOT in http://maven.imagej.net/content/groups/public was cached in the local repository, resolution will not be reattempted until the update interval of imagej.public has elapsed or updates are forced
